vscode/src/utils.ts

```diff
@@ -132,6 +119,41 @@ export class Context {
   }
 }
 
+class DeviceOutput {
+  context: Context;
+  device: Device;
+  childProcess?: cp.ChildProcess;
+  output?: OutputChannel;
+
+  constructor(ctx: Context, device: Device) {
+    this.context = ctx;
+    this.device = device;
+  }
+
+  start() {
+    if (this.childProcess) {
+      if (this.output) this.output.show(true);
+      return;
+    }
+
+    this.output = Window.createOutputChannel(`Toit (${this.device.name})`);
+    this.output.show(true);
+    this.childProcess = cp.execFile(this.context.toitExec, [ "dev", "-d", this.device.deviceID, "logs" ]);
```

Comment:
This seems to just show the "logs" of a device.
It seems like the original `run` and `deploy` output has been removed and replaced with the `logs`. But they aren't fully equivalent.

Reply:
In what respect are they not fully equivalent?

Reply:
The logs have all kinds of information. Not just the output of the `run`.
They might have log lines like
```
2021-06-21T10:29:38.976745Z: [kernel,network] INFO: connecting network=Wifi(toitwifi)
```
intermixed with the what the `run` command is doing.

If the device has already an app installed that is running at the same time as the `run`, then the output of this app would be mixed with the output of the `run`.

Reply:
We could have two different outputs, one for logs (used by deploy) and one for run. However, then we start having a lot of outputs, and I think that might be confusing as well.